src/components/Books.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function Books() {
+  return (
+
+    <div className="pt-5 container">
+      <h3>List of books</h3>
+      <button className="btn-dark btn-small rounded" type="button">Remove book</button>
+      <div className="d-flex pt-5">
+        <input type="text" placeholder="Add your book" />
+      </div>
+    </div>
```

Comment:
- [ ] You should render a list of books using the `Book` component, which doesn't exist right now. Let's create it.
- [ ] You should render a form using the `CreateNewBook` component, which doesn't exist now. Let's create it.

Reply:
This change has been done well